src/persistence/notification_logs.rs

```diff
@@ -0,0 +1,93 @@
+use log::debug;
+use sqlx::SqliteConnection;
+
+use super::get_default_sql_connection;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, sqlx::Type, Clone)]
+#[serde(rename_all = "lowercase")]
+pub enum NotificationType {
```

Comment:
single point of all types of notifications. when we need a new type, adding a variant here would be enough